bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TT_SStt/TT_SStt_run_card.dat

```diff
@@ -0,0 +1,255 @@
+
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+'lhapdf' = pdlabel     ! PDF set
+325500 = lhaid
+# To see heavy ion options: type "update ion_pdf"
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+False = fixed_ren_scale  ! if .true. use fixed ren scale
+False = fixed_fac_scale  ! if .true. use fixed fac scale
+91.188 = scale            ! fixed ren scale
+91.188 = dsqrt_q2fact1    ! fixed fact scale for pdf1
+91.188 = dsqrt_q2fact2    ! fixed fact scale for pdf2
+-1 = dynamical_scale_choice ! Choose one of the preselected dynamical choices
+1.0  = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Type and output format
+#*********************************************************************
+True     = gridpack  !True = setting up the grid pack DP EDIT: was False
+-1.0 = time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+average =  event_norm       ! average/sum. Normalization of the weight in the LHEF
+# To see MLM/CKKW  merging options: type "update MLM" or "update CKKW"
+
+#*********************************************************************
+#
+#*********************************************************************
+# handling of the helicities:
+#  0: sum over all helicities
+#  1: importance sampling over helicities
+#*********************************************************************
+0  = nhel          ! using helicities importance sampling or not.
+#*********************************************************************
+# Generation bias, check the wiki page below for more information:   *
+#  'cp3.irmp.ucl.ac.be/projects/madgraph/wiki/LOEventGenerationBias' *
+#*********************************************************************
+None = bias_module  ! Bias type of bias, [None, ptj_bias, -custom_folder-]
+{} = bias_parameters ! Specifies the parameters of the module.
+#
+#*******************************
+# Parton level cuts definition *
+#*******************************
+#
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay
+#*********************************************************************
+  15.0  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+ #*********************************************************************
+ # Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+ # (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+ #*********************************************************************
+   False  = cut_decays    ! Cut decay products
```

Comment:
since you are using decay chain, you need to change it to "True" if you want the following cuts to be applied on the decay objects